### DISCUSSION TITLE
Boleto Banco Sofisa

### DISCUSSION THREAD

**Discussion by alessandrobrunolima** [braully/bopepo#26] [Q&A] [answered]
Bom dia, estou precisando gerar boleto para o Banco Sofisa, gostaria de saber se alguém já implementou algo?

**Comment by braully** [comment, marked as answer]
Desconheço, até mesmo em outros forks do boleto original. Mas acredito que exista a quantidade de codigo suficiente para que você faça a customização, adicione a logo e veja os campos que compoem o codigo de barra, acredito que basta duplicar de outros bancos.

**Reply by alessandrobrunolima** [reply]
Bom dia, estou fazendo a implementação no meu fork, assim que finalizar crio o pull request para analise e teste. Obrigado
